en/sql/dblink.rst

```diff
@@ -642,23 +642,199 @@ The Query statement below is a Query statement that inquires the remote_t table
-* However, unlike SELECT queries, all functions used in INSERT/UPDATE/DELETE/MERGE queries execute on the remote server.
-* If you need to use a remote server function in a SELECT query, you can use the DBLINK statement.
-* Since the functions used in INSERT/UPDATE/DELETE/MERGE queries are executed on the remote server, care must be taken when using the functions. (In other words, when using CUBRID built-in functions, note that the remote DBMS may not have the corresponding built-in function or the usage may be different.)
-
-
-Restrictions
-=============================================
-
-*   DBLink for heterogeneous databases only supports utf-8.
-*   Only Unicode ODBC Driver must be used in the gateway.
-*   The maximum string length of one column is supported up to 16M.
-*   When using cache in Mysql, it is recommended to use PREFETCH, NO_CACHE=1 because the memory usage of the gateway cub_cas_cgw increases.
-*   ODBC non-supported types are SQL_INTERVAL, SQL_GUID, SQL_BIT, SQL_BINARY, SQL_VARBINARY, SQL_LONGVARBINARY.
-*   When using DBLink with heterogeneous types (Oracle/MySQL/MariaDB), you must use Oracle/MySQL/MariaDB's Unicode ODBC driver.
-*   When performing a query that includes the repeat() function in MySQL/MariaDB, part of the string may be cut off or the string may not be read.
+*   Synonym creation: Local synonyms can be created for remote tables and remote synonyms.
+
+.. code-block:: sql
+
+    -- for CUBRID
+    create synonym synonym_1 for t1@srv1;
+    create synonym synonym_2 for remote_synonym@srv1;
+
+    -- for ORACLE
+    create synonym synonym_ora for user_ora.t1@srv_ora;
+
+    -- for MySQL and MariaDB
+    create synonym synonym_my for my_db.t1@srv_mysql;
+    create synonym synonym_maria for maria_db.t1@srv_maria;
+
+*   Reserved word processing: When using a reserved word in the remote DB as an identifier, it must be enclosed in square brackets ([ ]) along with the reserved word processing characters in the remote DB.
+
+.. code-block:: sql
+
+    SELECT ["COLUMN"],["ADD"],["ALTER"] FROM ["TABLE"]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select "COLUMN","ADD","ALTER" from "TABLE" ') AS t(a varchar, b varchar, c varchar );
+
+*   The character used to use reserved words as identifiers in Oracle is the double quotation mark (" ") character.
+
+.. code-block:: sql
+
+    SELECT ["COLUMN"],["ADD"],["ALTER"] FROM ["TABLE"]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select "COLUMN","ADD","ALTER" from "TABLE" ') AS t(a varchar, b varchar, c varchar );
+
+*   The character used to use reserved words as identifiers in MySQL and MariaDB is the backquote (\` \`).
+
+.. code-block:: sql
+
+    SELECT [`COLUMN`],[`ADD`],[`ALTER`] FROM [`TABLE`]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select `COLUMN`,`ADD`,`ALTER` from `TABLE` ') AS t(a varchar, b varchar, c varchar );
+
+
+Constraints
+===========
+
+Common Constraints
+------------------
+
+*   The charset of the remote database used in DBLink supports only utf-8.

```

Comment:
How about changing as like the following ?
The charset of the remote database must be unicode (utf-8).

Reply:
Ok.